_test/page.getter.test.php

```diff
@@ -0,0 +1,32 @@
+<?php
+//======================================================================
+// PAGEGETTER TESTS
+//======================================================================
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../admin.php');
+require_once(__DIR__ . '/../_test/Helper.php');
+require_once(__DIR__ . '/../PageGetter.php');
+
+class page_getter_test extends DokuWikiTest
+{
+    public function setUp()
+    {
+        Helper::setUp();
+    }
+    public function test_get_pages_without_title_only_gets_pages_without_title()
```

Comment:
Test

* 0: Empty Wiki; Wiki where all pages have a title.
* 1: The test covers this already.
* Multiple: Multiple pages without a title.
* A lot: What if you have 10.000.000 pages without a title? Does this work? How does your UI handle this?